mantle/cmd/kola/switchkernel.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/coreos/mantle/kola"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdSwitchKernel = &cobra.Command{
+		RunE:    runSwitchKernel,
+		PreRunE: preRun,
+		Use:     "switch-kernel",
+		Short:   "Test on switching between Default and RT Kernel",
+	}
+
+	rtKernelRpmDir string
+)
+
+var (
+	homeDir            = `/var/home/core`
```

Comment:
Can I assume `core` user is always available in a `platform.Machine`?

Reply:
Yeah.